src/services/helpers/types.ts

```diff
@@ -28,16 +28,17 @@ export interface FactoryResolver {
 }
 
 export enum WorkspaceStatus {
-  RUNNING = 1,
-  STOPPING,
-  STOPPED,
-  STARTING,
-  PAUSED,
-  ERROR,
+  RUNNING = 'RUNNING',
+  STOPPING = 'STOPPING',
+  STOPPED = 'STOPPED',
+  STARTING = 'STARTING',
+  PAUSED = 'PAUSED',
+  ERROR = 'ERROR',
 }
 
 export enum DevWorkspaceStatus {
   FAILED = 'Failed',
+  STARTING = 'Starting',
```

Comment:
It seems to miss terminating state, and as result I assume Terminating workspaces are not displayed, to check that I've added some logs:
![Screenshot_20210427_121625](https://user-images.githubusercontent.com/5887312/116217523-6646b980-a752-11eb-8b98-dff23d7cd56d.png)

Also, note that after I removed Starting devworkspace, I got the warning in browser logs:
![Screenshot_20210427_121704](https://user-images.githubusercontent.com/5887312/116217625-7e1e3d80-a752-11eb-9d0a-00e551587bdd.png)

Reply:
I have fixed it